src/Sulu/Bundle/CoreBundle/Composer/ScriptHandler.php

```diff
@@ -17,6 +17,21 @@
 class ScriptHandler
 {
     const GIT_IGNORE_FILE = '.gitignore';
+    const ENV_FILE = '.env';
+
+    /**
+     * Copy the .env.dist file to .env file if not exists and generate a app secret.
+     */
+    public static function copyEnvDistFile()
+    {
+        if (file_exists(static::ENV_FILE)) {
+            return;
+        }
+
+        $envFile = file_get_contents(static::ENV_FILE . '.dist');
+        $envFile = str_replace('APP_SECRET=', 'APP_SECRET=' . bin2hex(random_bytes(16)), $envFile);
```

Comment:
inspired by the flex generate(secret)  function: https://github.com/symfony/flex/blob/5e6b06e0f452a895526873fa1a4531d2b209a098/src/Configurator/EnvConfigurator.php#L156